docs/installation/presto-port-configuration.rst

```diff
@@ -20,10 +20,6 @@ You can configure the server to use a different port by changing the following p
     discovery.uri=http://<coordinator_ip_or_host>:<port>
 
 You can add these properties with the new ``port`` to the file ``config.properties`` under each of the directories ``/etc/opt/prestoadmin/coordinator``
-and ``/etc/opt/prestoadmin/workers`` locally, prior to running :ref:`server-install-label`. You may need to create these two directories. 
-If you have already installed Presto using :ref:`server-install-label` then you
-can reconfigure the cluster to use a new port by updating the ``config.properties`` at  ``/etc/opt/prestoadmin/coordinator``
+and ``/etc/opt/prestoadmin/workers`` locally. You should then reconfigure the cluster to use a new port
+by updating the ``config.properties`` at  ``/etc/opt/prestoadmin/coordinator``
```

Comment:
I would give the full paths as ``/etc/opt/prestoadmin/coordinator/config.properties`` and ``/etc/opt/prestoadmin/workers/config.properties``